deepchem/models/chemberta.py

```diff
@@ -0,0 +1,83 @@
+import torch
```

Comment:
Can you put these imports into an import guard? Like in https://github.com/deepchem/deepchem/pull/1967

Reply:
Will do! My intention right now is to get a working version with all the packages (which will be put into an import guard), before then putting them into an import guard